Observer/AdyenHppDataAssignObserver.php

```diff
@@ -23,16 +23,27 @@
 
 namespace Adyen\Payment\Observer;
 
+use Adyen\Payment\Observer\Adminhtml\AdyenSateDataValidator;
 use Magento\Framework\Event\Observer;
-use Magento\Quote\Api\Data\PaymentInterface;
 
 class AdyenHppDataAssignObserver extends AdyenAbstractDataAssignObserver
 {
     /**
-     * Approved root level keys from additional data array
-     *
-     * @var array
+     * @var \Adyen\Service\Validator\CheckoutStateDataValidator
      */
+    public $checkoutStateDataValidator;
+
+    public $adyenHelper;
+
+    public function __construct(
+        \Adyen\Service\Validator\CheckoutStateDataValidator $checkoutStateDataValidator,
+        \Adyen\Payment\Helper\Data $adyenHelper
```

Comment:
```suggestion
        \Adyen\Service\Validator\CheckoutStateDataValidator $checkoutStateDataValidator
```
please remove $adyenHelper if not used